### PR TITLE
[PM-2237] Add tailwind to desktop & browser

### PR DESCRIPTION
**Pull request by Hinton**
## Type of change

```
- [ ] Bug fix
- [x] New feature development
- [x] Tech debt (refactoring, code cleanup, dependency upgrades, etc)
- [ ] Build/deploy pipeline (DevOps)
- [ ] Other
```

## Objective

Adds support for using Tailwind along with the component library in the browser and desktop application. Do note that it's currently not possible to share higher tier components in the component library since `libs/components` depends on `libs/angular`.

## Code changes

Adds `postcss.config.js` to the clients, updates webpack to use postcss, and adds a tailwind css file.

## Before you submit

- Please add **unit tests** where it makes sense to do so (encouraged but not required)
- If this change requires a **documentation update** - notify the documentation team
- If this change has particular **deployment requirements** - notify the DevOps team
- Ensure that all UI additions follow [WCAG AA requirements](https://contributing.bitwarden.com/contributing/accessibility/)

### CODE REVIEW

File: libs/components/src/search/search.component.css [hunk 1]
```diff
@@ -9,8 +9,8 @@ bit-search input[type="search"]::-webkit-search-cancel-button {
   margin: 0;
   cursor: pointer;
   background-repeat: no-repeat;
-  mask-image: url("./close-button-white.svg");
-  -webkit-mask-image: url("./close-button-white.svg");
```
Comment (willmartian) [inline libs/components/src/search/search.component.css:13]: Why is this necessary?

**Review by willmartian — APPROVED** [review]
Looks good! 🥇 

One question:

File: libs/components/src/search/search.component.css (hunk 1 above)
Comment (Hinton) [reply libs/components/src/search/search.component.css:13]: To resolve a name conflict.

**Review by MGibson1 — APPROVED** [review]
@Hinton we need to move most of these files under the platform team.

Do you see a clear way to add that to the code owners file? It's be nice not to have to move too much.

**Comment by willmartian** [comment]
> @Hinton we need to move most of these files under the platform team.
> 
> Do you see a clear way to add that to the code owners file? It's be nice not to have to move too much.

@MGibson1 Does this suffice?
```CODEOWNERS
postcss.config.js @bitwarden/team-platform-dev
tailwind.css @bitwarden/team-platform-dev
tailwind.config.js @bitwarden/team-platform-dev
tsconfig.json @bitwarden/team-platform-dev
webpack.config.js @bitwarden/team-platform-dev
```
